docs/DEV_ENVIRONMENT.md

```diff
@@ -38,6 +38,11 @@
        ```
 9. Try `cargo test --all --features run-docker-tests -- --test-threads=16`.
 
+   Note for MacOS users:
+    - The UTXO 'testblockchain' docker container may fail to start due to TCP port conflict with MacOS AirPlay Receiver (using the same 7000 port). Disabling AirPlay Receiver resolves this problem.
```

Comment:
Why are we not changing the container port instead of telling users to disable AirPlay :thinking:

Reply:
We could suggest the user to change the port by themselves, as another option

Reply:
Why not changing the container port directly? That way users don't have to change or know anything about it.

Reply:
We may change it ofc, no guarantee though that there would not conflicts with other services.
We may rephrase the note as more general like: if containers do not start check for port conflicts, for example for MacOS it may be a conflict with AirPlay